cmd/parse.go

```diff
@@ -9,39 +9,21 @@ import (
 )
 
 func Parse(programName string, args []string, fn func(flags *flag.FlagSet)) {
-
-	var versionFlag bool
-
 	flags := flag.NewFlagSet(programName, flag.ContinueOnError)
 	flags.SetOutput(flag.CommandLine.Output())
-	flags.BoolVar(&versionFlag, "version", false, "Show version information")
 
 	fn(flags)
 
-	switch programName {
-	case "ibmcloud":
-		if len(args) < 10 {
-			flags.PrintDefaults()
-			Exit(1)
-		}
-	case "aws":
-		if len(args) < 2 {
-			flags.PrintDefaults()
-			Exit(1)
-		}
-
+	if len(args) == 0 {
+		flags.PrintDefaults()
+		Exit(1)
 	}
 
-	if err := flags.Parse(args[1:]); err != nil {
+	if err := flags.Parse(args); err != nil {
```

Comment:
Please keep to use `args[1:]` to suppress the unit test failures. `flags.Parse` assumes `args[0]` is a program name.